src/group_actions/group_action_interface.jl

```diff
@@ -0,0 +1,213 @@
+#
+#
+# Group Action related types and functions
+
+@doc """
+    AbstractGroupActionType
+
+An abstract supertype for group actions.
+"""
+abstract type AbstractGroupActionType end
+
+_note_action_argument_order = """
+!!! note
+   In function definitions we follow the idea of the family of actions and use the order `(M, g, p)` in function signatures.
```

Comment:
```suggestion
    In function definitions we follow the idea of the family of actions and use the order `(M, g, p)` in function signatures.
```
This text didn't render in the box but below it (probably one space missing?)